proto/dwtools/abase/l0/l8/gBuffer.s

```diff
@@ -2625,6 +2625,37 @@ function bufferResize_( dst, srcBuffer, size )
 
 
 //
+  /**
+   * The bufferBytesGet() routine converts and returns a new instance of Uint8Array constructor( array of 8-bit unsigned integers ).
+   *
+   * @param { BufferRaw|BufferNode|BufferTyped|string } src - Defined buffer or string.
+   *
+   * @example
+   * let src2 = new BufferRaw( 5 );
+   * _.bufferBytesGet(src2);
+   * // returns [ 0, 0, 0, 0, 0, ]
+   *
+   * @example
+   * let src1 = BufferNode.alloc( 5, 'a' );
+   * _.bufferBytesGet(src);
+   * // returns [ 97, 97, 97, 97, 97 ]
+   *
+   * @example
+   * let src3 = new I32x( [ 5, 6, 7 ] );
+   * _.bufferBytesGet(src3);
+   * // returns [ 5, 0, 6, 0, 7, 0 ]
+   *
+   * @example
+   * let src4 = 'Hello World';
```

Comment:
Please, use short string in example
`Hello World` -> `string`
or
`Hello World` -> `abcd`